python/spear/tools.py

```diff
@@ -8,9 +8,15 @@
 # command-line arguments, so we specify them in DefaultGame.ini.
 def get_cook_dirs():
     return [
+        os.path.join("Content", "__ExternalActors__"),
+        os.path.join("Content", "__ExternalObjects__"),
         os.path.join("Content", "Characters"),
+        os.path.join("Content", "LevelPrototyping"),
         os.path.join("Content", "Spear"),
-        os.path.join("Content", "StarterContent")]
+        os.path.join("Content", "StarterContent"),
+        os.path.join("Content", "ThirdPerson"),
+        os.path.join("Content", "Vehicles"),
+        os.path.join("Content", "VehicleTemplate")]
 
 def get_cook_maps():
```

Comment:
I think we need to add the maps we're using in your examples here, otherwise we won't be able to access them in PAK builds.